org.eclipse.jdt.ls.product/scripts/jdtls.py

```diff
@@ -0,0 +1,93 @@
+import argparse
```

Comment:
As this is a source file (particularly one we'll be shipping from Eclipse servers (eg. https://download.eclipse.org/jdtls/milestones/1.8.0/, https://repo.eclipse.org/content/repositories/jdtls-releases/org/eclipse/jdt/ls/org.eclipse.jdt.ls.product/1.8.0.20220126150144/org.eclipse.jdt.ls.product-1.8.0.20220126150144.tar.gz), we should add a copyright header here much like we do for Java sources (obviously the comment declaration will be a bit different, with `"""`).

See https://www.eclipse.org/projects/handbook/#ip-copyright-headers for the specific requirement.

Otherwise, change looks good to me.

Reply:
I've add the headers, hopefully in the correct way. Can you double check?

Reply:
Yes, just update the years to 2022 and `SPDX-License-Identifier` should be `EPL-2.0` (no 3 at end) as per https://spdx.org/licenses/EPL-2.0.html .

Reply:
Sorry, stupid copy paste error on my side. 😅 Fixed.